site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -544,3 +543,36 @@
 </script>
 
 
+<script>
+    window.onload = function() {
+        // capture radio elements
+        let radioBulk = document.getElementById('radio-bulk');
+        let radioNormal = document.getElementById('radio-normal');
+        let radioStudent = document.getElementById('radio-student');
+        let numericUploadOpts = document.getElementById('numeric-split-opts');
+        let qrUploadOpts = document.getElementById('qr-split-opts');
+
+        // capture the div elements
+        let pdfSubmitButton = document.getElementById('pdf-submit-button');
+
+        // add change event listeners to radio buttons
+        radioBulk.addEventListener('change', togglePdfSubmitButton);
+        radioNormal.addEventListener('change', togglePdfSubmitButton);
+        radioStudent.addEventListener('change', togglePdfSubmitButton);
+
+        function togglePdfSubmitButton() {
+            if (radioBulk.checked) {
+                // if Bulk Upload is checked, show the div
+                pdfSubmitButton.style.display = 'block';
+                qrUploadOpts.style.display = 'none';
+
+            } else {
+                // if other option is checked, hide the div
+                pdfSubmitButton.style.display = 'none';
+            }
+        }
+
+        // call the function on page load to set initial state
+        togglePdfSubmitButton();
```

Comment:
It would be better to set the initial state via Twig, to prevent a flash as the JS runs initially.